static/js/factories/embedly.js

```diff
@@ -27,6 +27,23 @@ export const makeImage = () => ({
   type:          "photo"
 })
 
+export const makeTweet = () => ({
+  provider_url:    "http://twitter.com",
```

Comment:
Should this be `https` or does it not really matter?

Reply:
this is just a sample response I captured from embedly, so yeah it should be https but...

in the test it's really just a placeholder so I think it's ok